src/app/my-skills/my-skills.component.html

```diff
@@ -1 +1,75 @@
-<p>my-skills works!</p>
+<div class="basic container-fluid">
+    <div class="row justify-content-center">
+        <div class="title col-2">
+            <div class="section-heading">
+                <div class="line"></div>
+                <h2>My <span>Skills</span></h2>
+            </div>
+        </div>
+            
+        <div class="skills col-9">
+            <div class="skill">
```

Comment:
@Yuriy-27  Why did you not use *ngFor for skills? please update
u can store skill array in the component or better in service (just to imitate that data is being retrieved from BE)